CommonClient.py

```diff
@@ -502,10 +502,11 @@ async def send_msgs(self, msgs: typing.List[typing.Any]) -> None:
         """ `msgs` JSON serializable """
         if not self.server or not self.server.socket.open or self.server.socket.closed:
             return
-        await self.server.socket.send(encode(msgs))
+        await self.server.socket.send(encode_to_bytes(msgs))
```

Comment:
I am wondering if this is a breaking change. Could be that the old call would send it as Text frame and the new call would send it as Binary frame. Did you check?